dietpi/func/dietpi-banner

```diff
@@ -245,65 +248,77 @@ $GREEN_LINE"
+			(( ${aENABLED[17]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[17]} $GREEN_SEPARATOR $(free -b | mawk 'NR==2 {CONVFMT="%.0f"; print $3/1024^2" of "$2/1024^2" MiB ("$3/$2*100"%)"}')"
+			# Load average
+			(( ${aENABLED[18]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[18]} $GREEN_SEPARATOR $(mawk '{print $1 ", " $2 ", " $3}' /proc/loadavg) ($(nproc) cores)"
+			# Hostname
+			(( ${aENABLED[3]} && ! ${aENABLED[14]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[3]} $GREEN_SEPARATOR $(</etc/hostname)"
+			# NIS/YP domainname
+			(( ${aENABLED[4]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[4]} $GREEN_SEPARATOR $(hostname -y 2>&1)"
+			# LAN IP
+			Print_Local_Ip
+			# WAN IP + location info
+			(( ${aENABLED[6]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[6]} $GREEN_SEPARATOR $(G_GET_WAN_IP 2>&1)"
+			# DietPi-VPN connection status
+			(( ${aENABLED[13]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[13]} $GREEN_SEPARATOR $(/boot/dietpi/dietpi-vpn status 2>&1)"
+			# Disk usage (RootFS)
+			(( ${aENABLED[7]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[7]} $GREEN_SEPARATOR $(df -h --output=used,size,pcent / | mawk 'NR==2 {print $1" of "$2" ("$3")"}' 2>&1)"
+			# Disk usage (DietPi userdata)
+			(( ${aENABLED[8]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[8]} $GREEN_SEPARATOR $(df -h --output=used,size,pcent /mnt/dietpi_userdata | mawk 'NR==2 {print $1" of "$2" ("$3")"}' 2>&1)"
+			# Weather
+			(( ${aENABLED[9]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[9]} $GREEN_SEPARATOR $(curl -sSfLm 3 'https://wttr.in/?format=4' 2>&1)"
+			# Let's Encrypt cert status
+			if (( ${aENABLED[16]} ))
+			then
+				echo -en "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[16]} $GREEN_SEPARATOR "
+				if [[ $EUID == 0 ]]
+				then
+					# Running as root
+					Print_Cert_Status 2>&1
+				else
+					# Running as non-root: Fail silently without NOPASSWD to avoid password prompt, but print info instead
+					sudo -n dash -c "$(declare -f Print_Cert_Status); Print_Cert_Status 2>&1" 2> /dev/null || echo 'NOPASSWD sudo required to obtain cert status'
+				fi
+			fi
+			# Custom
+			(( ${aENABLED[10]} )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[10]} $GREEN_SEPARATOR $(bash "$FP_CUSTOM" 2>&1)"
+			# MOTD
+			if (( ${aENABLED[12]} ))
+			then
+				local motd fp_motd='/run/dietpi/.dietpi_motd'
+				[[ -f $fp_motd ]] || curl -sSfLm 3 'https://dietpi.com/motd' -o "$fp_motd"
+				# shellcheck disable=SC1090
+				[[ -f $fp_motd ]] && . "$fp_motd" &> /dev/null && [[ $motd ]] && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[12]} $GREEN_SEPARATOR $motd"
+			fi
+			echo -e "$GREEN_LINE\n"
 
-		(( ${aENABLED[15]} )) && Print_Credits
-		Print_Updates
-		(( ${aENABLED[11]} )) && Print_Useful_Commands
+			(( ${aENABLED[15]} )) && Print_Credits
+			Print_Updates
+			(( ${aENABLED[11]} )) && Print_Useful_Commands
+		) | ${FOLD_CMD}
+		echo "fold_cmd=${FOLD_CMD}"
```

Comment:
Use for debugging, please remove this line after testing/verifying that all seems correct